src/Microsoft.AspNetCore.BuildTools.ApiCheck/build/Microsoft.AspNetCore.BuildTools.ApiCheck.targets

```diff
@@ -0,0 +1,30 @@
+﻿<!--
```

Comment:
Done, mainly because consuming the Api Check package directly may be useful in some cases e.g. just to run the tool from a command line. Don't want to affect the consuming project then.